api/src/modules/scenario-interventions/dto/create.scenario-intervention.dto.ts

```diff
@@ -170,7 +170,7 @@ export class CreateScenarioInterventionDto {
     enum: Object.values(LOCATION_TYPES),
     example: LOCATION_TYPES.POINT_OF_PRODUCTION,
   })
-  newLocationType?: LOCATION_TYPES;
+  newLocationType!: LOCATION_TYPES;
```

Comment:
Sorry, I keep forgetting to change LOCATION_TYPES enum to have only the 4 location types we agreed on with Science ( UNKNOWN, AGGREGATION_POINT, POINT_OF_PRODUCTION, COUNTRY_OF_PRODUCTION), maybe we can do it in this PR to have this change sooner?

Reply:
Done! -> 5f1f110e90966c6427ac9603d8bec99b92bc92e9